python/ray/air/config.py

```diff
@@ -649,9 +650,7 @@ class RunConfig:
     verbose: Optional[Union[int, "AirVerbosity", "Verbosity"]] = None
     stop: Optional[Union[Mapping, "Stopper", Callable[[str, Mapping], bool]]] = None
     callbacks: Optional[List["Callback"]] = None
-    progress_reporter: Optional[
-        "ray.tune.progress_reporter.ProgressReporter"  # noqa: F821
-    ] = None
+    progress_reporter: Optional["ray.tune.progress_reporter.ProgressReporter"] = None
```

Comment:
```suggestion
    progress_reporter: Optional["ProgressReporter"] = None
```

Reply:
This would completely undo what was changed by @can-anyscale in https://github.com/ray-project/ray/pull/47138#discussion_r1717592244. My guess is that this is needed for sphinx. Changing it back I guess would break it. Though I don't know how to check.

Reply:
@can-anyscale it would be good to hear from you here.